EngD/test_config_in_ff_sync.R

```diff
@@ -0,0 +1,20 @@
+code_location <- "C:/Kodingan3/ForestForesight/"
+library("devtools")
+load_all(code_location) # with this we are using the live R code, not the build package of ForestForesight
+
+download_folder <- "C:/Kodingan3/FFdata/" #adjust this
```

Comment:
are these comments necessary? # adjust this, or # does this work?

Reply:
OK, actually I can change the download folder var into using config file